cfme/infrastructure/virtual_machines.py

```diff
@@ -160,6 +160,46 @@ def in_infra_vms(self):
             self.navigation.currently_selected == ['Compute', 'Infrastructure', 'Virtual Machines'])
 
 
+class OrphanedVmsAllView(InfraVmView):
+    actions = View.nested(CompareToolBarActionsView)
```

Comment:
These widgets are the same on these two new Views, and `VmsTemplatesAllView`. Its the same view, with different navigation destinations leading to different accordion selections.

The duplication here needs to be squashed, either through subclassing, or implementing `am_i_here` for the navigation destinations, instead of several view classes with _very_ similar is_displayed methods.

Since this is new automation, instead of a small 'fix', and given our release cycle, I'm considering this required changes.

There are just a few small things that we can do here, in order to flatten these view/nav classes, and eliminate all this duplication.